theme/mixins/backdrop.less

```diff
@@ -1,11 +1,15 @@
-.backdrop() {
+.transparentBackdrop() {
   position: fixed;
   top: -999px;
   left: -999px;
   right: -999px;
   bottom: -9999px;
-  background-color: @sk-black;
-  opacity: 0.7;
   transition: opacity 0.3s;
   z-index: @z-index-inline-overlay;
 }
+
+.shadedBackdrop() {
```

Comment:
we can't, because there will be a name clash. Alternatively we can rename the file and then it should be fine, let's call it backdrops.less then?